src/data/parts.js

```diff
@@ -0,0 +1,82 @@
+ import { parts, cars } from "./init.js";
+import { ObjectId } from "mongodb";
+import Validator from "../utils/validator.js";
+import { DataBaseException, NotFoundException } from "../utils/exceptions.js";
+
+const createPart = async (name, price, manufacturer, sellerId, carIds) => {
+    const partData = Validator.validatePart({
+        name,
+        price,
+        manufacturer,
+        sellerId,
+        carIds
+    });
+
+    let newPart = {
+        name: partData.name,
+        price: partData.price,
+        manufacturer: partData.manufacturer,
+        sellerId: new ObjectId(partData.sellerId),
+        carIds: partData.carIds.map(id => new ObjectId(id)),
+        dateAdded: new Date()
+    };
+
+    try {
+        const partsCollection = await parts();
+        const insertInfo = await partsCollection.insertOne(newPart);
+        if (!insertInfo.acknowledged || !insertInfo.insertedId) {
+            throw new Error('Could not add part');
+        }
+        const newId = insertInfo.insertedId;
+        return await getPartById(newId.toString());

```

Comment:
don't query the database, just return: 
{...newPart, _id: insertInfo.insertedId}